app/views/faq.scala.html

```diff
@@ -3,17 +3,17 @@
 @main("FAQ", isLoggedIn = isLoggedIn) {
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-l">Transfer Digital Records Frequently Asked Questions</h1>
+            <h1 class="govuk-heading-l">Transfer Digital Records<br/>Frequently Asked Questions</h1>
```

Comment:
Should we just call the page `Frequently Asked Questions` so we don't need the line break? The name of the service is in the header at the top of the page.

Reply:
I'd be more than happy to just call it Frequently Asked Questions.

I've looked up guidance on using <br> as well and was looking at the same source as you which reassured me though that it is OK to use it in this context. Maybe I got that wrong (?), but I much prefer to shorten the headline anyway.

Reply:
Hmm, maybe I was misreading that section. Do you think it only applies to paragraphs of text?

I think the main reason that `<br>` in the header looks odd to me is that we're using to change the appearance. But how does that look at different screen sizes or zoom levels?

Reply:
It is certainly not ideal which I think it is good to have a shorter title :)